sites/partners/styles/overrides.scss

```diff
@@ -59,3 +59,13 @@
     }
   }
 }
+
+.alert-box.primary {
+  background-color: var(--bloom-color-gray-400);
+}
+
+#sign-in {
```

Comment:
This is a Big Sad but this has caused a lot of spinning and is blocking other work